sources/packages/web/src/views/supporting-user/Login.vue

```diff
@@ -1,40 +1,46 @@
 <template>
-  <v-card elevation="2" class="mx-auto mt-15" max-width="690px" outlined>
-    <v-card-header>
-      <v-card-header-text>
-        <v-card-title class="my-6 bold-text" style="font-size: 27px"
-          >Welcome to Student Application Supporting Information
-          Portal</v-card-title
-        >
-        <v-card-subtitle>Welcome text goes here…</v-card-subtitle>
-      </v-card-header-text>
-    </v-card-header>
-    <v-card-text
-      >We are using BCSC for authentication. Please click on Login/Register
-      buttons below to start your sign in/sign up.</v-card-text
-    >
-    <v-row class="m-3">
-      <v-col>
-        <v-btn
-          justify-center
-          color="primary"
-          @click="login"
-          class="float-right"
-          prepend-icon="fa:fa fa-user"
-        >
-          Login with BCSC
-        </v-btn>
-      </v-col>
-      <v-col>
-        <v-btn
-          color="primary"
-          @click="login"
-          prepend-icon="mdi-account-star-outline"
-        >
-          Sign Up with BCSC
-        </v-btn>
-      </v-col>
-    </v-row>
+  <v-card elevation="2" class="mx-auto mt-15" max-width="730px">
```

Comment:
Not a part of the PR. For all the login pages we can create a container and make a common pattern with slots.

Reply:
yes we can, as all login mostly follows the same template, like a login-container or something.

Reply:
is the pattern same for all login pages?

Reply:
Yes, I think so. 

![image](https://user-images.githubusercontent.com/54600590/195948963-be4f072e-3530-4d90-8f9a-5a2bbc5dc07f.png)

![image](https://user-images.githubusercontent.com/54600590/195949033-f2acc58a-efe2-41f9-aced-692c27fa9594.png)

![image](https://user-images.githubusercontent.com/54600590/195949107-c45c9357-7f13-4845-9d99-47f77a03c6ff.png)

![image](https://user-images.githubusercontent.com/54600590/195949217-c64ef9dc-79ee-4525-b2b9-db3c950397ef.png)

Reply:
Okay, but it doesn't look like a container, because if you see a student, there is another card below. but for the upper card. we can create a structure.

Reply:
Yes @dheepak-aot, I agree that they are similar enough to a component be created. We can have a tech debit since all login pages are currently changed. We can take care of this on an upcoming opportunity.

Reply:
Created one https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1460